src/pages/Home/index.tsx

```diff
@@ -115,21 +115,14 @@ export default () => {
   const appContext = useContext(AppContext)
 
   useEffect(() => {
```

Comment:
I think the cause of many requests is that useEffect should be one-time shoot here, setBlocksWrappers/ setStatistics /appContext should not be the direct dependence. especially here appContext cotain error message(if one request failed, this listener would reenter)